dolphinscheduler-storage-plugin/dolphinscheduler-storage-hdfs/src/main/java/org/apache/dolphinscheduler/plugin/storage/hdfs/HdfsStorageOperator.java

```diff
@@ -182,7 +182,7 @@ public Configuration getConfiguration() {
      * @return DefaultFS
      */
     public String getDefaultFS() {
-        String defaultFS = getConfiguration().get(Constants.FS_DEFAULT_FS);
+        String defaultFS = getConfiguration().get(Constants.HDFS_DEFAULT_FS);
```

Comment:
We should find a way to compatible with `resource.hdfs.fs.defaultFS`.

Reply:
Hi, @SbloodyS , if defaultFS here is null, we'll read it from `resource.hdfs.fs.defaultFS`, may i ask what do you mean a compatible way?

Reply:
> Hi, @SbloodyS , if defaultFS here is null, we'll read it from `resource.hdfs.fs.defaultFS`, may i ask what do you mean a compatible way?

Both `resource.hdfs.fs.defaultFS` and `core-site.xml` and `hdfs-site.xml` can be used normally.

Reply:
Sounds good to me. @kingbabingge

Reply:
tks. I commit the code.

> Sounds good to me. @kingbabingge

Reply:
tks. I commit the code.